plugins/shortcuts/src/api/LocalStoredShortcuts.ts

```diff
@@ -25,7 +25,11 @@ import { StorageApi } from '@backstage/core-plugin-api';
  * Implementation of the ShortcutApi that uses the StorageApi to store shortcuts.
  */
 export class LocalStoredShortcuts implements ShortcutApi {
-  constructor(private readonly storageApi: StorageApi) {}
+  constructor(private readonly storageApi: StorageApi) {
+    this.storageApi.observe$<Shortcut[]>('items').subscribe({
+      next: () => this.notify(),
+    });
+  }
 
   shortcut$() {
     return this.observable;
```

Comment:
Maybe we can actually replace this internal observable with a mapping of the one from the storage API? Something like `Observable.from(this.storageApi.observe$<Shortcut[]>('items')).map(snapshot => snapshot.value ?? [])`. That way we could get rid of a lot of code.